policykit/scripts/starterkits.py

```diff
@@ -30,9 +30,18 @@
     "fail": "pass",
 }
 
+all_actions_execute = {
+    "filter": "return True",
+    "initialize": "pass",
+    "check": "return PASSED",
+    "notify": "pass",
+    "success": "action.execute()",
+    "fail": "pass",
+}
+
 testing_policy1_slack = GenericPolicy.objects.create(
-    **all_actions_pass,
-    description="Starter Constitution Policy: all constitution actions pass automatically",
+    **all_actions_execute,
+    description="Starter Constitution Policy: all constitution actions execute automatically",
```

Comment:
@rickygv99 can you explain why these starterkit policies need to call `action.execute()`?